### ISSUE TITLE
Optimization Trajectories

### ISSUE THREAD

**Issue by Edoardo-Pedicillo** [ElStabilini/optimization#7]
During the fine-tuning of parameters, both through the optimizer and _by hand_ , it may be useful to track the gate fidelity (or sequence fidelity). Since this is already integrated into the optimizer’s loss function, it should be extended to manual tuning as well.

we could visualize the results as points on a frequency-amplitude(-drag) space. The color of each point would represent the gate (or sequence) fidelity, providing a clear visual representation ( see for example https://how2matplotlib.com/matplotlib-color-scales.html).

This approach could offer valuable insights into how the two tuning methods explore the parameter space and whether certain regions are inaccessible.

Probably this issue is similar to #4

**Comment by alecandido** [comment]
> we could visualize the results as points on a frequency-amplitude(-drag) space. The color of each point would represent the gate (or sequence) fidelity, providing a clear visual representation ( see for example [how2matplotlib.com/matplotlib-color-scales.html](https://how2matplotlib.com/matplotlib-color-scales.html)).

Agreed: I'd just avoid 3D plots, since they may look fancy, but they are 2D projections, and when they are not interactive, and especially not clean enough, the information is easily lost...

But we can allow making different 2D projections, even on arbitrary planes (it's just a matter of basic linear algebra).
